utils.py

```diff
@@ -1047,6 +1061,42 @@ def get_latest_release_url(releases_url):
     return release_url
 
 
+def get_latest_release_version_tag_name(releases_url):
+    data = net_get(releases_url)
+    if data:
+        try:
+            json_data = json.loads(data)
+            logging.debug(f"{json_data=}")
+        except json.JSONDecodeError as e:
+            logging.error(f"Error decoding JSON response: {e}")
+            return None
+
+        if not isinstance(json_data, list) or len(json_data) == 0:
+            logging.error("Invalid or empty JSON response.")
+            return None
+    else:
+        logging.critical("Could not get latest release URL.")
+        return None
+
+    release_tag_name = json_data[0].get('tag_name')  # noqa: E501
+    logging.info(f"Release URL Tag Name: {release_tag_name}")
+    return release_tag_name
+
+
+def set_logoslinuxinstaller_latest_release_config():
+    releases_url = "https://api.github.com/repos/FaithLife-Community/LogosLinuxInstaller/releases"  # noqa: E501
+    logoslinuxinstaller_url = get_latest_release_url(releases_url)
+    logoslinuxinstaller_tag_name = get_latest_release_version_tag_name(releases_url)
+    if logoslinuxinstaller_url is None:
+        logging.critical("Unable to set LogosLinuxInstaller release without URL.")  # noqa: E501
+        return
+    config.LOGOS_LATEST_VERSION_URL = logoslinuxinstaller_url
+    config.LOGOS_LATEST_VERSION_FILENAME = os.path.basename(logoslinuxinstaller_url) #noqa: #501
+    # Getting version relies on the filename having this format:
+    #   vXXXX
+    config.LLI_LATEST_VERSION = logoslinuxinstaller_tag_name.lstrip('v')
```

Comment:
I'd say that this depends on the *tag_name* in the URL JSON data rather than the filename, right?